catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -376,13 +412,19 @@ protected void printStatus(long count) {
             return null;
         }
 
-        List<MetacardTransformer> metacardTransformerList = new ArrayList<MetacardTransformer>();
+        List<MetacardTransformer> metacardTransformerList = new ArrayList<>();
```

Comment:
Can you replace this method by wiring a ReferenceList of transformers into the command?  That would eliminate the need to access the bundleContext directly.